modules/services/mopidy.nix

```diff
@@ -113,7 +123,13 @@ in {
         After = [ "network.target" "sound.target" ];
       };
 
-      Service = { ExecStart = "${mopidyEnv}/bin/mopidy"; };
+      Service = {
+        ExecStart = "${mopidyEnv}/bin/mopidy --config ${
+            concatStringsSep ":"
+            ([ "${config.xdg.configHome}/mopidy/mopidy.conf" ]
+              ++ cfg.extraConfigFiles)
```

Comment:
Can extract to a variable (inside the `let` bindings at the top), then use the variable both here and below.

Reply:
Should I extract the entire `concatStringsSep` call or only extract `[ "${config.xdg.configHome}/mopidy/mopidy.conf" ] ++ cfg.extraConfigFiles`? Thank you for the review!

Reply:
I would say the entire thing, so that the command in both places contains simply something like `--config ${configFilePaths}`.